dart-lang/googleapis
# merge discovery common

- Initial commit
- First version of the common discovery APIs package.
- Prepare for publish.
- Expose the Escaper class which is also used by the generated library code.
- Add DetailedApiRequestError and mapping from JSON
- Publish new version of the discovery_commons package.
- fix strong mode issues
- Rev the pubspec version to capture DDC fixes (https://github.com/dart-lang/discoveryapis_commons/pull/5).
- Remove crypto dependency and upgrade dart sdk dependency
- move to pkg/test
- dartfmt
- fix imports
- Upgrade version in pubspec.yaml
- Make everything strong-mode clean (#8)
- Prepare for new version, finish strong-mode cleaning and add 'analysis_options.yaml' file
- Fix USES_DYNAMIC_AS_BOTTOM violations (#10)
- Updates for Dart 2.0 core library changes (wave 2.2)
- add travis-ci support
- Fix strong mode issue with Dart2 (#13)
- Fix screaming consts. (#14)
- Fix range check. (#15)
- Dart 2 fix (#16)
- Prepare release. (#17)
- Fix test in Dart 2 mode. (#19)
- dartfmt
- test: cleanup deprecated usages
- analysis: disable implicit casts, enable other lints
- Prepare to release 0.1.6+1
- misc: fix analyzer error regarding missing return value (#21)
- Allow string error codes in error responses
- Save jsonResponse in exception
- Address review
- Fix responses where the content-length isn't given (#24)
- Support latest pkg:http, prepare for release (#26)
- Fix new analyzer errors
- Enable and fix lints via dartfmt --fix
- fix some spelling error
- dartfmt fixes
- Remove forbidden headers
- Send client library identifier
- Prepare a new release
- Ignore uri not exists
- Make `ApiRequestError` implement `Exception` and not `Error`.
- Update pubspec.yaml for null safety.
- Migrate to null safety.
- Update CHANGEOG.md, pubspec.yaml, and .travis.yaml.
- Fix lints and enable CI
